### PR TITLE
Update README to reflect release of Spark 3.2

### PR DESCRIPTION
**Pull request by nightscape**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,14 @@ Add the following dependency to your `build.sbt`:
...
 
 As Scala 3 can use Scala 2.13 libraries, your Spark-Scala3 project needs a Spark
-version published for 2.13. As of now, Spark 3.2 is scheduled to be compatible
-with Scala 2.13 and the Apache Spark team publishes a Snapshot version of 3.2.
+version published for 2.13. Spark 3.2+ is compatible with Scala 2.13.
 
 You can add it in your `build.sbt` with:
 
 ```scala
 ThisBuild / resolvers += "Spark Snapshots" at "https://repository.apache.org/content/repositories/snapshots"
```
Comment (vincenzobaz) [inline README.md:22]: This line can be removed

**Review by vincenzobaz — CHANGES_REQUESTED** [review]
Thank you @nightscape for the quick reaction!

We can drop the addition of the snapshot resolver.
Could you also adapt line 11 to reflect the new version number?

**Comment by nightscape** [comment]
@vincenzobaz like this?

**Review by vincenzobaz — APPROVED** [review]
Thank you!

**Pull request merged**